repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+/content/taxonomies:
```

Comment:
We currently follow `snake_case` (e.g. `advanced_critical_care_practitioners`) for keys of value-lists. Could we stick to it for taxonomy category keys as well for consistency?

Also, could we follow the sentence case for the taxonomy category label (`locale.en.name`) to be in line with value-list labels ? It is possible that the taxonomy spread sheet might be little outdated and so could we cross-check the taxonomy categories against corresponding existing value-lists like `/content/documents/administration/valuelists/publicationtypes`, `/content/documents/administration/valuelists/publicationtopics` and `/content/documents/administration/valuelists/publicationprofessions` ?